scripts/jenkins/mkcloud/openstack-mkcloud.builder.sh

```diff
@@ -184,13 +184,18 @@ export clouddescription="Jenkins Build: $BUILD_NUMBER / $JOB_NAME"
 
 $custom_settings
 
-perl -e "alarm 6*60*60 ; exec '${mkcloudwrapper} bash ${automationrepo}/scripts/mkcloud setuphost $(echo -n $MKCLOUDTARGET) ' ; print qq{$!\n} ; exit 127 " | tee $artifacts_dir/mkcloud_short_stdout.log
+perl -e "alarm 7*60*60 ; exec '${mkcloudwrapper} bash ${automationrepo}/scripts/mkcloud setuphost $(echo -n $MKCLOUDTARGET) ' ; print qq{$!\n} ; exit 127 " | tee $artifacts_dir/mkcloud_short_stdout.log
 ret=${PIPESTATUS[0]}
 if [[ $ret != 0 ]] ; then
     if [[ $github_pr_sha ]] ; then
         mkcloudgating_trap
     fi
-    echo "mkcloud ret=$ret"
+    # code 142 means process exited because of SIGALRM
+    if [[ $ret == 142 ]]; then
+        echo "mkcloud stopped because of timeout. ret=$ret"
```

Comment:
Please also write the current value of timeout

Reply:
I extracted the timeout to a variable.